### ISSUE TITLE
On DMN Runner table, loading the inputs will not update the SelectField component

### ISSUE THREAD

**Issue by ljmotta** [apache/incubator-kie-issues#230]
Loading the inputs of a DMN should update the fields correctly. If an inputs file has a SelectField value, it will not visually update it after the load, it's necessary to close and open the DMN Runner table.

### Steps to reproduce
 - Open the DMN Sample;
 - Open the DMN Runner table;
 - Set a value for the "Martial Status" field;
 - Under the "Run" button, use the "Download inputs" option;
 - Change the "Martial Status" to another value;
 - Under the "Run" button, use the "Load inputs" option, and use the recently downloaded file;
 - The "Martial Status" will not be visually updated. Close and open the DMN Runner table to see the correct value;

### Example

https://user-images.githubusercontent.com/24302289/234107962-e6fb7c4e-e425-47d3-ace9-4e88a5bd1559.mp4

### Code
After a "Load inputs" operation is necessary to force a re-render of the SelectField or if this is not possible, the entire table. These changes will take place under the `unitables` package. The cell component is the `UnitablesBeeTableCell` under the `UnitablesBeeTable.tsx` file. Other possible components that this change could take place is the `Unitables` or the `UnitablesBeeTable`.

```[tasklist]
### Acceptance criteria
- [ ] "SelectField" value should be visually updated after the "Load inputs" operation;
```

**Comment by jomarko** [comment]
Here are my findings. During debugging I noticed this behavior in `DmnRunnerContextProvider.tsx`

There is this code block, executed on `Load inputs` menu item is selected. We can see it also from first line - comment of the block
```js
        // CFSF_ADD is triggered on file creation, file rename, persistence deletion or upload.
        if (workspaceFileEvent.type === "CFSF_ADD") {
          const dmnRunnerPersistenceJson = dmnRunnerPersistenceService.parseDmnRunnerPersistenceJson(
            workspaceFileEvent.content
          );

          // Remove incompatible values and add default values;
          const jsonSchemaProperties = getObjectValueByPath(jsonSchema, JSON_SCHEMA_PROPERTIES_PATH);
          if (jsonSchemaProperties) {
            Object.entries(jsonSchemaProperties).forEach(
              ([propertyName, dmnField]: [string, DmnInputFieldProperties]) => {
                dmnRunnerPersistenceJson.inputs.forEach((input) => {
                  // if the input has the property name, its value should match the property attribute type/format;
                  if (input[propertyName] && typeof input[propertyName] !== (dmnField.type ?? "string")) {
                    input[propertyName] = getFieldDefaultValue(dmnField);
                  }
                });
              }
            );
          }
```

Furthermore, the same file `DmnRunnerContextProvider.tsx`, contains a code block `setDmnRunnerInputs`, that is executed, when tabular view of the DMN Runner is opened. Basically 12th second from the video attached to description of #230. However `setDmnRunnerInputs` is not executed when `Load inputs` dropdown menu item is selected.

```js
  const setDmnRunnerInputs = useCallback(
    (newInputsRow: ((previousInputs: Array<InputRow>) => Array<InputRow>) | Array<InputRow>) => {
      setDmnRunnerPersistenceJson({ newInputsRow });
    },
    [setDmnRunnerPersistenceJson]
  );
```

**Comment by ljmotta** [comment]
Indeed, this is the code that is executed when the `Load inputs` is selected. And we don't have a problem in this part of the code. You can check this by loading inputs in a normal field:

https://github.com/kiegroup/kie-issues/assets/24302289/0054b013-00a2-4455-96e3-4fab60c26f7c

Loading the inputs doesn't trigger a **visual** update in the SelectField, but change its value, as we can obseve when we close and open the drawer. So what we need to do is to trigger this visual update in the SelectField.

**Comment by ljmotta** [comment]
The [kie-tools#1666](https://github.com/kiegroup/kie-tools/pull/1666) addressed this issue.

**Comment by jomarko** [comment]
@ljmotta thank you for the kiegroup/kie-tools#1666. May I ask you to explain a little bit more what part of that PR fixed #230? Was that fix in `UnitablesBeeTaleCell` as you wrote in issue description, or did the fix landed different place?

Also just side question, why do we promote kogito jira instead kie-issues?
